packages/cli/src/errors/response-errors/abstract/response.error.ts

```diff
@@ -1,9 +1,9 @@
-import { ApplicationError } from 'n8n-workflow';
+import { UnexpectedError } from 'n8n-workflow';
 
 /**
  * Special Error which allows to return also an error code and http status code
  */
-export abstract class ResponseError extends ApplicationError {
+export abstract class ResponseError extends UnexpectedError {
```

Comment:
Hmmm I think `ResponseError` should actually inherit `BaseError`, because some responses are due to user (most 4xx), some are transient (502-504) and some unexpected (all other 5xx). So we should set the `level` based on the http code